scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -1460,10 +1526,16 @@ static u32 task_pick_domain(struct task_ctx *taskc, struct task_struct *p,
 			 */
 			if (first_dom == NO_DOM_FOUND)
 				first_dom = dom;
+
+			if (has_preferred_dom < 0)
+			       continue;
+
+			if (((1LLU << dom) & taskc->preferred_dom_mask))
```

Comment:
Should we also check `&& preferred_dom != NO_DOM_FOUND`?

Reply:
Hmm @hodgesds, looks like we still need this check?

Reply:
I double checked this, since `preferred_dom` is initialized to `NO_DOM_FOUND` it would always fail that check I think.

Reply:
reading comprehension > me.... yeah that's good!